dex-ws-load/src/test/resources/gatling.conf

```diff
@@ -0,0 +1,127 @@
+#########################
```

Comment:
All options are commented. Why do we need this file?

Reply:
deleted